### PR TITLE
renaming compliance log directory

### PR DESCRIPTION
**Pull request by georgelyuan**
renaming from /audit to /compliance to avoid confusion

### CODE REVIEW

**Comment by georgelyuan** [comment]
@christ1ne can this be approved? Or tagged for discussion next meeting?

File: submission_rules.adoc [hunk 1]
```diff
@@ -297,19 +297,22 @@ Training benchmark directory names must be one of  { **resnet, ssd, maskrcnn, tr
...
+**** <benchmark>/
+***** <scenario>
+****** <test_id>
+******* performance/
+******** run_1/ # 1 run for every scenario
+********* mlperf_log_summary.txt
+********* mlperf_log_detail.txt
+******* accuracy/
+******** accuracy.txt # for TEST01 only, generated from truncate_accuracy_log.py
+******** mlperf_log_accuracy.json
```
Comment (nvpohanh) [inline submission_rules.adoc:311]: @georgelyuan Is this file needed for other tests?

File: submission_rules.adoc [hunk 2]
```diff
@@ -297,19 +297,22 @@ Training benchmark directory names must be one of  { **resnet, ssd, maskrcnn, tr
...
-******** mlperf_log_detail.txt
-****** accuracy/
-******* mlperf_log_accuracy.json
-****** <verify_performance.py stdout>
-****** <verify_accuracy.py stdout>
+** compliance/
+*** <system_desc_id>/
+**** <benchmark>/
+***** <scenario>
+****** <test_id>
```
Comment (nvpohanh) [inline submission_rules.adoc:304]: @georgelyuan Could you list out the accepted values for this below? I think the accepted values are `TEST01` `TEST04-A` `TEST04-B` and `TEST05`.

**Comment by TheKanter** [comment]
Reviewed on 9/1, conflict, George to resolve conflict and clear at 9/8 meeting.

**Review by nvpohanh — APPROVED** [review]

File: submission_rules.adoc [hunk 3]
```diff
@@ -299,27 +299,31 @@ Training benchmark directory names must be one of  { **resnet, ssd, maskrcnn, tr
...
+******** compliance_accuracy.txt # only for TEST01 if accuracy check fails
+******* verify_performance.txt
+******* verify_accuracy.txt # for TEST01 only
 
 
 System names and implementation names may be arbitrary. 
 
-Inference benchmark directory names must be one of  { **ssd-mobilenet, resnet50, ssd-resnet34, rnnt, bert-99, bert-99.9, dlrm-99, dlrm-99.9, 3d-unet-99, 3d-unet-99.9
- **}. The postfix '-99' and '-99.9' indicate that the accuracy must be >= 99% or 99.9% of the target accuracy.
+<benchmark> must be one of {**resnet, ssd-small, ssd-large, rnnt, bert-99, bert-99.9, dlrm-99, dlrm-99.9, 3d-unet-99, 3d-unet-99.9**}. The postfix '-99' and '-99.9' indicate that the accuracy must be >= 99% or 99.9% of the target accuracy.
```
Comment (nvpohanh) [inline submission_rules.adoc:322]: @georgelyuan Please fix `ssd-mobilenet, resnet50, ssd-resnet34` names. Thanks
